src/config.js

```diff
@@ -13,7 +13,7 @@ const contextPath = (typeof process !== 'undefined' && typeof process.env !== 'u
   typeof process.env.REACT_APP_SERVER_CONTEXT_PATH !== 'undefined' && process.env.REACT_APP_SERVER_CONTEXT_PATH !== null) ?
   process.env.REACT_APP_SERVER_CONTEXT_PATH : window.location.href.split('/')[3];
 
-const apiBaseUrl = `${serverAddress}${contextPath}/ws/rest/v1`;
+const apiBaseUrl = `/${serverAddress}${contextPath}/ws/rest/v1`;
```

Comment:
Yes, I am looking to add the `/` because the URL of any API request being made from the `react-component` is always appended to the base URL of it's parent repo. I ended up having request like this `http://localhost:3000/openmrs-standalone/owa/openmrs-owa-orderentry/openmrs-standalone/ws/rest/v1/appui/session`, so I had to specify the trailing `/` to ensure it does not append to any base URL of it's parent repo. so now my API requests are like this `http://localhost:3000/openmrs-standalone/ws/rest/v1/appui/session`

Reply:
Great, I think it will work in that case... there may end up being a double "/" in some cases, but I don't think that will be a problem.